model/src/main/scala/uk/gov/ons/addressIndex/model/db/index/PostcodeAddressFile.scala

```diff
@@ -0,0 +1,89 @@
+package uk.gov.ons.addressIndex.model.db.index
+
+import uk.gov.ons.addressIndex.model.server.response.{PAF, PAFWithFormat}
+
+trait Formattable {
+  def delimitByComma(parts: String*) = parts.map(_.trim).filter(_.nonEmpty).mkString(", ")
+}
+
+case class PostcodeAddressFile(
```

Comment:
Maybe we should move these into
uk.gov.addressIndex.model.db.index.format to provide context?